### ISSUE TITLE
Baseline Accessibility Audit

### ISSUE THREAD

**Issue by Sequoia** [hernansartorio/jquery-nice-select#44]
This plugin works visually, but fails in complying with many a11y requirements. For example a user that navigates primarily by keyboard will have to press down 180 times to select Uganda. In order to get this plugin at minimum ADA / ARIA compliance, please implement the following:

- [ ] typeahead navigation
- [ ] aria-* attributes for better form navigation and screen reader support
